src/material/stepper/step-header.scss

```diff
@@ -28,11 +28,13 @@
     &:hover:not([aria-disabled]),
     &:hover[aria-disabled='false'] {
       @include token-utils.create-token-slot(background-color, header-hover-state-layer-color);
+      @include token-utils.create-token-slot(border-radius, header-hover-border-radius);
     }
 
     &.cdk-keyboard-focused,
     &.cdk-program-focused {
       @include token-utils.create-token-slot(background-color, header-focus-state-layer-color);
+      @include token-utils.create-token-slot(border-radius, header-hover-border-radius);
```

Comment:
This is using a hover token for the focused state. In other places we have two separate tokens.

Reply:
good catch! I added it now